pkg/virtctl/console/console.go

```diff
@@ -131,3 +144,70 @@ func (c *Console) Run(args []string) error {
 	}
 	return nil
 }
+
+// AttachConsole attaches stdin and stdout to the console
+// in -> stdinWriter | stdinReader -> console
+// out <- stdoutReader | stdoutWriter <- console
+func (c *Console) attachConsole(message string, resChan <-chan error) (err error) {
```

Comment:
I don't see the removal from `pkg/virtctl/utils` as part of this commit or PR?

Reply:
@lyarwood 

Because it is using by other commands 
I will remove it Eventually but for now I want to make sure my refactoring of console command is correct

Reply:
@aliamerj still, code duplication are bad, so I don't think we should have a copy of that code here

Reply:
okay , where should i put it ??
in the issue description 

`Move AttachToConsole here from pkg/virtctl/utils`